.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
 
-before_install:
-    - wget http://bit.ly/miniconda -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - conda update --yes conda
-    - travis_retry conda create --yes -n test $CONDA --file requirements.txt
-    - source activate test
-    - travis_retry conda install --yes pip
-    - travis_retry pip install pytest
+# tdk: enable master when merged
+branches:
+  only:
+    - flexible-mesh
+#    - master
 
+sudo: false
 
 install:
-    - mkdir files
+  - if [[ "$CONDA_PYTHON_VERSION" == "2" ]]; then
+      wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    fi
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  # Useful for debugging any issues with conda
+  - conda info -a
+  - yes | conda install conda-build jinja2
+  - conda config --add channels nesii/channel/ocgis
+
+  # Build and install.
+  - pushd conda-recipe
+  - conda build .
+  - yes | conda install `conda build . --output`
+  - yes | conda install pytest
+  - popd
```

Comment:
The lines were added to use conda build. Looking at it again, it could be simplified somewhat. If you don't want to use conda build, then this code is unnecessary.